eol/alpine/eof.go

```diff
@@ -0,0 +1,109 @@
+package alpine
+
+import (
+	"bytes"
+	"fmt"
+	"github.com/PuerkitoBio/goquery"
+	"github.com/aquasecurity/vuln-list-update/utils"
+	"github.com/spf13/afero"
+	"golang.org/x/xerrors"
+	"log"
+	"os"
+	"path/filepath"
+	"strings"
+	"time"
+)
+
+const (
+	eolAlpineFolder = "eol/alpine"
+	eolAlpineFile   = "eol-alpine"
```

Comment:
What if we have an entire file name here?
```suggestion
	eolAlpineFile   = "alpine.json"
```

Reply:
Changed